### ISSUE TITLE
[Suggestion] Biome Grouping

### ISSUE THREAD

**Issue by SnowBlitzz** [Pixelmon-Development/BetterDexRewards#40]
![image](https://user-images.githubusercontent.com/7488382/211512546-00e1d72b-317f-4f7e-a3c6-b16b9ecd7ae6.png)
To prevent overflow of biomes listed in unregistered pokemon gui, especially when biome mods are added such as BYG or BOP, maybe a config option to group the biomes into biome groups would make sense. Similar to what's in the betterspawning config, as an example, everything under mountainous will just be displayed as `mountainous biomes` then they can click on it to get the real list via /wiki command

    "mountainous": [
      "minecraft:mountains",
      "minecraft:mountain_edge",
      "minecraft:wooded_mountains",
      "minecraft:gravelly_mountains",
      "minecraft:modified_gravelly_mountains",
      "pixelmon:ultra_rocky",
      "pixelmon:ultra_crater",
      "pixelmon:ultra_cliff",
      "biomesoplenty:alps",
      "biomesoplenty:alps_foothills",
      "biomesoplenty:highland",
      "biomesoplenty:highland_crag",
      "biomesoplenty:highland_moor",
      "biomeoplenty:jade_cliffs",
      "byg:alpine_foothills",
      "byg:alps"

**Comment by danorris709** [comment]
Perhaps worth re-visiting in 1.20.1 with biome tags? Thoughts? @SnowBlitzz

**Comment by SnowBlitzz** [comment]
> Perhaps worth re-visiting in 1.20.1 with biome tags? Thoughts? @SnowBlitzz

yeah sounds good. I think there can even be some way to make it "expand" by if they need the full biome list
